HelpSource/Guides/WritingClasses.schelp

```diff
@@ -284,9 +285,11 @@ methods should take only one argument to support both ways of expression consist
 
 code::
 MyClass {
-    variable_ { | newValue |
+
+	variable_ { | newValue |
         variable = newValue.clip(minval,maxval);
```

Comment:
@telephon there is a mix of spaces and tabs in this method declaration and other few ones